component/loki/source/journal/types.go

```diff
@@ -0,0 +1,37 @@
+package journal
+
+import (
+	"time"
+
+	"github.com/grafana/agent/component/common/loki"
+	flow_relabel "github.com/grafana/agent/component/common/relabel"
+)
+
+// Arguments are the arguments for the component.
+type Arguments struct {
```

Comment:
Are we missing the `matches` field here?

Reply:
Oddly enough matches is not defined in the promtail configuration https://grafana.com/docs/loki/latest/clients/promtail/configuration/#journal

Reply:
Oh, probably an omission? It is mentioned on the scraping [page](https://grafana.com/docs/loki/latest/clients/promtail/scraping/#journal-scraping-linux-only) example as well as the JournalTargetConfig struct.

Reply:
I saw it on the config itself but didnt want to add it since it wasnt defined in the doc, but since the scraping page mentions it then we should.